web/package.json

```diff
@@ -11,7 +11,7 @@
     "build": "npm run scaffold && NODE_ENV=production webpack -p",
     "lint": "eslint 'src/**/*.js'",
     "start": "webpack-dev-server --config webpack.config.dev.js",
-    "test": "jest --collectCoverage",
+    "test": "TZ=utc jest --collectCoverage",
```

Comment:
Set the timezone so the human-readable thingy will be consistent when running tests, regardless of the test machine's actual timezone.